src/Uno.Extensions.Templates/content/unoapp-extensions/build/steps/mobile.yml

```diff
@@ -0,0 +1,75 @@
+parameters:
+- name: major
+  type: string
+  default: '0'  
+- name: minor
+  type: string
+  default: '0'
+- name: package_id
+  type: string
+  default: 'release_mobile'
+- name: project_to_build
+  type: string
+  default: ''
+- name: project_folder
+  type: string
+  default: ''         
+- name: target_framework
+  type: string
+  default: 'net7.0-android'
```

Comment:
This default value doesn't make much sense, set this to '' or don't specify a value to make it a required parameter